log-window.js

```diff
@@ -26,261 +26,87 @@ class EmeLogWindow {
   }
 
   /** Open the log window. */
```

Comment:
I got confused by something about the architecture here, and it's not anything you introduced.  But would you please change the fileoverview above to say something like "Creates and communicates with the actual log window"?

The design of this was messy, and if anything, V3 forces us to have a cleaner separation between what lives in the pop-up window (log.js) and what lives in the extension (log-window.js).  So if you could update that comment, I think that would help.  I'll consider renaming the file later, but for now, I think not renaming it will make the diff easier to read.

Reply:
Thanks for the suggestion! I agree with you, I'll focus on the migration to V3 this time. We can refactor this as needed in the future. I've updated the file overview with "Creates and communicates with the actual log window."